src/GraphQL/Schemas/Primary/Mutations/SetWalletAccountMutation.php

```diff
@@ -50,7 +50,7 @@ public function args(): array
                 'type' => GraphQL::type('Int'),
                 'description' => __('enjin-platform::query.get_wallet.args.id'),
                 'rules' => [
-                    'prohibits:externalId',
+                    'required_without:externalId',
```

Comment:
Should we leave the `prohibits` in the rules so one or the other is required, but not both, at the moment I think this rule might allow both to be passed in?
```suggestion
                    'prohibits:externalId,required_without:externalId',
```